addons/project/static/src/views/project_task_list/project_task_list_controller.js

```diff
@@ -0,0 +1,40 @@
+/** @odoo-module */
+
+import { ListController } from "@web/views/list/list_controller";
+import { DeleteSubtasksConfirmationDialog } from "@project/components/delete_subtasks_confirmation_dialog/delete_subtasks_confirmation_dialog";
+import { sprintf } from "@web/core/utils/strings";
+import { session } from "@web/session";
+
+export class ProjectTaskListController extends ListController {
+    async onDeleteSelectedRecords() {
+        if (!Math.max(...this.model.root.selection.map(function(record) { return record.data.subtask_count }))) {
```

Comment:
Transformed it to an anonymous function + as it is a oneliner, deleted brackets and "return". :wink: 
```suggestion
        if (!Math.max(...this.model.root.selection.map((record) => record.data.subtask_count))) {
```